src/session/content/send_media_window.rs

```diff
@@ -33,19 +26,26 @@ mod imp {
     }
 
     #[glib::object_subclass]
-    impl ObjectSubclass for SendPhotoDialog {
-        const NAME: &'static str = "ContentSendPhotoDialog";
-        type Type = super::SendPhotoDialog;
+    impl ObjectSubclass for SendMediaWindow {
+        const NAME: &'static str = "ContentSendMediaWindow";
+        type Type = super::SendMediaWindow;
         type ParentType = adw::Window;
 
         fn class_init(klass: &mut Self::Class) {
             klass.bind_template();
 
             klass.install_action_async(
-                "send-photo-dialog.send-message",
+                "send-media-window.send-message",
                 None,
                 |widget, _, _| async move {
-                    widget.send_message().await;
+                    widget.send_message(false).await;
+                },
+            );
+            klass.install_action_async(
+                "send-media-window.send-as-file",
+                None,
+                |widget, _, _| async move {
+                    widget.send_message(true).await;
                 },
             );
```

Comment:
I was investigating whether we could use a single action like
```rust
klass.install_action_async(
    "send-media-window.send-message",
    Some("b"),
    |widget, _, variant| async move {
        widget.send_message(variant.unwrap().get().unwrap()).await;
    },
);
```

But it seems the MenuItem's `target` isn't recognized as a bool
```
menu more_options_menu {
  section {
    item {
      label: _("Send as _File");
      action: "send-media-window.send-message";
      target: "true";
    }
  }
}
```

Do you have an idea? Otherwise, let's just keep it as it is.

Reply:
No idea, I tried this already with the message.delete (for the revoke option), but I ended up creating a separate action for the same reason.